addons/mail/static/src/models/suggested_recipient_info.js

```diff
@@ -23,6 +23,16 @@ registerModel({
         _computeName() {
             return this.partner && this.partner.nameOrDisplayName || this.name;
         },
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeTitleText() {
+            return this.env._t(_.str.sprintf(
```

Comment:
How about using the `sprintf` function from `@web/core/utils/strings`?

Reply:
Changes done (and also reorder `sprintf(this.env._t())`